app/components/landscape-portrait-slider.js

```diff
@@ -1,32 +1,242 @@
+                        <h2 className={classes.tailoredHeader}>{portraitHeader}</h2>
+                        <p className={classes.tailoredParagraph}>{portraitParagraph}</p>
+                    </div>
+                </div>
+
+                <div className={classes.daContainer}>
+                    <SvgRectangle className={classes.iconda} />
+                    <p className={classes.iconda}> {daLink}</p>
+                    <p className={classes.icondaPerson}> Windspotter, Rabiechel</p>
+                </div>
+
+                <div className={classes.iconContainer}>
+                    <i
+                        onClick={handelInputDesktop}
+                        className={cx(!isLandscape && classes.noborderBottom, isLandscape && classes.borderBottom)}
+                    >
+                        <SvgDesktop className={classes.icon} />{' '}
+                    </i>
+                    <i
+                        onClick={handelInputPortrait}
+                        className={cx(isLandscape && classes.noborderBottom, !isLandscape && classes.borderBottom)}
+                    >
+                        <SvgPortrait className={classes.icon} />
+                    </i>
+                </div>
+            </div>
         </>
     )
 }
 
 //possibly set height to a ratio based on width (w/1920*1080)
 const useStyles = createUseStyles(theme => ({
+    body: {
+        backgroundColor: 'white',
+        maxWidth: '100%',
+    },
+    noborderBottom: {
+        cursor: 'pointer',
+    },
+    tailoredHeader: {
+        fontSize: '1.4rem',
+    },
+    tailoredParagraph: {
+        fontSize: '1rem',
+    },
+    tailoredContainer: {
+        width: '50%',
+        position: 'relative',
+        marginLeft: '1rem',
+        marginTop: '20%',
+    },
+    portraitContainer: {
+        width: '50%',
+        position: 'relative',
+        overflow: 'hidden',
+        justifyContent: 'center',
+        margin: 'auto',
+        left: '0%',
+    },
+    portraitTailoredContainer: {
+        display: 'flex',
+        flexDirection: 'row',
+    },
+    borderBottom: {
+        cursor: 'pointer',
+        borderBottom: '#7470FF solid 0.3rem',
+    },
+    disabled: {
+        display: 'none',
+    },
+    landscapeContainer: {
+        position: 'relative',
+        overflow: 'hidden',
+        width: '100%',
+        paddingTop: '56.25%',
+    },
     landscape: {
+        position: 'absolute',
+        top: '0',
+        right: '0',
+        bottom: '0',
+        left: '0',
         width: '100%',
-        height: 'auto',
-        title: 'Used for testing',
+        height: '100%',
+    },
+    portrait: {
+        position: 'relative',
+        margin: 'auto',
+        flex: 'auto',
+        alignSelf: 'center',
+        justifyContent: 'center',
+        alignContent: 'center',
+        alignItems: 'center',
+        verticalAlign: 'center',
+        width: '20.2 rem',
```

Comment:
there can not be a space between 20.2 and rem.  It's throwing an error in the CSS.  Check for other instances.